Makefile

```diff
@@ -53,7 +53,7 @@ clean: ## clean the repository
 
 dev_install: ## set up the repo for active development
 	${PIP} install -e .[dev]
-	${PYTHON} -m jupyter serverextension enable --py jupyterfs
+	${PYTHON} -m jupyter serverextension enable --py jupyterfs.extension
```

Comment:
can this be done as:

```
dev_install: install serverextension js  ## set up the repo for active development
```

Reply:
dunno. My make-fu is not particularly advanced. I can check

Reply:
so your ultimate goal here is to make it so that the line

```
${PYTHON} -m jupyter serverextension enable --py jupyterfs.extension
```

appears only once in the Makefile, right? I just pushed a PR that makes it so. I had to tweak your suggestion a bit, but I was able to get something similar working and organized reasonably well.